src/fides/api/ops/schemas/application_config.py

```diff
@@ -28,6 +30,46 @@ def validate_storage_type(cls, storage_type: StorageType) -> StorageType:
         return storage_type
 
 
+# TODO: the below models classes are "duplicates" of the pydantic
+# models that drive the application config. this is to allow every field
+# to be optional on the API model, since we want PATCH functionality.
+# ideally, we'd not need to duplicate the config modelclasses, and instead
+# just make all fields optional by default for the API models.
+
+
+class NotificationApplicationConfig(BaseSchema):
+    """
+    API model - configuration settings for data subject and/or data processor notifications
+    """
+
+    send_request_completion_notification: Optional[bool]
+    send_request_receipt_notification: Optional[bool]
+    send_request_review_notification: Optional[bool]
+    notification_service_type: Optional[str]
+
+    @validator("notification_service_type", pre=True)
+    @classmethod
+    def validate_notification_service_type(cls, value: Optional[str]) -> Optional[str]:
+        """Ensure the provided type is a valid value."""
+        if value:
+            valid_values = ["MAILGUN", "TWILIO_TEXT", "TWILIO_EMAIL"]
```

Comment:
Slightly better performance with a tuple since the valid values are immutable. 

```suggestion
            valid_values = ("MAILGUN", "TWILIO_TEXT", "TWILIO_EMAIL")
```